indexer/database/bridge.go

```diff
@@ -0,0 +1,144 @@
+package database
+
+import (
+	"database/sql"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/jackc/pgtype"
+	"gorm.io/gorm"
+)
+
+/**
+ * Types
+ */
+
+type Transaction struct {
+	fromAddress common.Address
+	toAddress   common.Address
+	amount      pgtype.Numeric
+	data        []byte
+}
+
+type TokenPair struct {
+	l1TokenAddress common.Address
+	l2TokenAddress common.Address
```

Comment:
Nice this is useful to index using l2Contract.l1Address() (likely different name) as source of truth